src/python/pants/binaries/binary_tool.py

```diff
@@ -115,6 +166,22 @@ def register_options(cls, register):
       version_registration_kwargs['fingerprint'] = True
     register('--version', **version_registration_kwargs)
 
+    register('--version-digest-mapping', type=dict,
```

Comment:
How would you set one of these in a pants.ini file or on the command line? Are tuples representable in ini?

Reply:
Yes: we parse python literals from the `ini`.